pkg/conversion/test/fuzz/fuzz.go

```diff
@@ -440,6 +442,9 @@ func CAPIMachineFuzzerFuncs(providerIDFuzz StringFuzzer, infraKind, infraAPIVers
 			func(m *capiv1.Machine, c fuzz.Continue) {
 				c.FuzzNoCustom(m)
 
+				m.Annotations = sharedAnnotations()
+				m.Labels = sharedCAPILabels(clusterName)
+
```

Comment:
You could fuzz these normally and then set the MachineTemplateSpec lower. The MachineTemplateSpec fuzzer has already run by this point. Then you'd still be fuzzing labels and would only test static labels/annotations

Reply:
Yeah you are right, updated.